consume-lsp

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# This script is meant to be run after the Flux release script has
+# opened a PR in the LSP, and that PR has merged.
+
+# It should handle the rest of the release process, with minimal 
+# babysitting by an engineer.
+
+# If any command fails, exit the script
+set -e
+
+# Check if hub is installed
+if ! command -v hub &> /dev/null
+then
+    echo "hub is not installed. exiting"
+fi
+
+# Make a temporary directory where we can clone fresh copies of all the necessary repos
+TEMPDIR=$(mktemp -d -t lsp-release.XXXX)
+echo "Using ${TEMPDIR}"
+cd $TEMPDIR
+
+# Cut down on code duplication
+function tmp_clone() {
+	git clone git@github.com:influxdata/$1.git &> /dev/null
+	echo "$(pwd)/$1"
+}
+
+function extract_version() {
+	v=$(echo "$1" | grep -Po -m 1 '\d+\.\d+\.\d+')
+	echo $v
+}
+
+FLUX_DIR=$(tmp_clone flux)
+LSP_DIR=$(tmp_clone flux-lsp)
+VSFLUX_DIR=$(tmp_clone vsflux)
+CLI_DIR=$(tmp_clone flux-lsp-cli)
+INFLUXDB_DIR=$(tmp_clone influxdb)
+
+# ui is a private repo, so maybe we should treat it like idpe,
+# in the sense that we don't want to expose its existence to the public.
+UI_DIR=$(tmp_clone ui)
+
+cd $FLUX_DIR
+flux_version=$(extract_version $(git describe))
+
+# TODO:
+# (maybe?) Confirm that the flux version matches what's in flux-lsp's Cargo.toml
```

Comment:
It would be rare, but if there's ever a case where there's been a flux release, but the new version hasn't made it into the LSP, we would want to exit this script.